apps/condo/migrations/20220923171250-0175_alter_ticketsource_type.js

```diff
@@ -0,0 +1,30 @@
+// auto generated by kmigrator
```

Comment:
Regenerate migrations

Reply:
Why? It's good migration index. Maybe later if is changed index in master